src/components/Book.js

```diff
@@ -0,0 +1,85 @@
+import React, { Component } from 'react';
+// eslint-disable-next-line import/no-extraneous-dependencies
+import PropTypes from 'prop-types';
+import { Link } from 'react-router-dom';
+
+class Book extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      genre: props.genre,
+      title: props.title,
+      author: props.author,
+      progress: 0,
+      currentChapter: 1,
+    };
+  }
+
+  handleComment = () => {
+    // code to handle adding comments
+  }
+
+  handleRemove = () => {
+    // code to remove the book
+  }
+
+  handleEdit = () => {
+    // code to edit the book
+  }
+
+  handleLoad = () => {
+    // code to load the progress of the book
+  }
+
+  render() {
+    const {
+      genre, title, author, progress, currentChapter,
+    } = this.state;
```

Comment:
- [ ] Like the previous reviewer mentioned, please remove the state from this Book component. There is no need to receive the title, genre, etc as props, then initialize each as state values, and then destructure of the state. Instead, you need to destructure the props directly and use them when rendering where it is needed. 
- [REQUIRED] You need to remove the state from this component.